### DISCUSSION TITLE
页码下标距离正文过近的问题

### DISCUSSION THREAD

**Discussion by SiriusFang2020** [sikouhjw/gdutthesis#47] [Q&A] [answered]
当正文文字较多并出现图片时，会导致页码下标与正文文字的距离过于接近，请问该如何解决？
![image](https://user-images.githubusercontent.com/59632817/158382886-cf9ba245-d929-4a89-9211-6196fbaaf912.png)

**Comment by sikouhjw** [comment, marked as answer]
在导言区写
```tex
\geometry{
  footskip    = 0.85cm,
}
```
原本的距离设置的不合理，现在这个距离是参考《广东工业大学研究生学位论文撰写规范格式模板》的页脚底端距离算出来的（2.6 - 1.75 = 0.85）。

**Reply by SiriusFang2020** [reply]
多谢，已解决🙏
